src/table.rs

```diff
@@ -75,6 +78,33 @@ impl Table {
     }
 }
 
+/// Open an iceberg table by uri
+pub async fn open(uri: impl AsRef<str>) -> Result<Table> {
+    // Todo(xudong): inferring storage types by uri
+    let mut builder = Fs::default();
+    builder.root(uri.as_ref());
+
+    let op = Operator::new(builder)?
+        .layer(LoggingLayer::default())
+        .finish();
+
+    let mut table = Table::new(op);
+    table.load().await?;
```

Comment:
load should be called by user itself so that they can decide when to load or load again.

Reply:
For the `open` semantic, I think should call `load` once, or the table doesn't be opened in fact.

Reply:
Makes sense.